app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,35 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+body {
+padding-top: 60px;
+}
```

```suggestion
body {
 padding-top: 60px;
}
```

check lại indent file này